idea_town/experiments/models.py

```diff
@@ -21,6 +21,7 @@ class Experiment(models.Model):
     version = models.CharField(blank=True, max_length=128)
     changelog_url = models.URLField(blank=True)
     contribute_url = models.URLField(blank=True)
+    addon_id = models.CharField(max_length=500, default='addonid@mozilla.com')
```

Comment:
Every addon published or signed by AMO has an ID, so while it should be required, I don't think we should provide a default.